plenigo_plugin/PlenigoContentManager.php

```diff
@@ -151,22 +153,35 @@ public function add_scripts() {
     public function plenigo_js_snippet() {
         PlenigoSDKManager::get()->getPlenigoSDK();
         $isPaywalled = $this->plenigo_paywalled_content();
-        /*
-          echo '<script type="application/javascript">'
-          . 'var plenigo = plenigo || {};'
-          . 'plenigo.baseURI = "' . PLENIGO_SVC_URL . '";'
-          . 'plenigo.baseStaticURI = "' . PLENIGO_SVC_URL . '";</script>'; */
 
+        if (DEF_PLENIGO_SVC_URL != PLENIGO_SVC_URL) {
```

Comment:
We are talking about global constants, The main reason for this if is to have code run if the endpoint is our test endpoint.